dtos/common/base.go

```diff
@@ -69,6 +69,21 @@ func NewBaseWithIdResponse(requestId string, message string, statusCode int, id
 	}
 }
 
+// BaseWithTotalCountResponse defines the base content for response DTOs (data transfer objects).
+// This object and its properties correspond to the BaseWithTotalCountResponse object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-data/2.x#/BaseWithTotalCountResponse
```

Comment:
please replace all the `2.x` to `2.1.0` from the swagger version, because we don't have 2.x anymore

Reply:
updated with https://github.com/edgexfoundry/go-mod-core-contracts/pull/670/commits/94dcb764258f71b83f82d8441e3f7c09de7f2e9f